### PR TITLE
Corrections to Rallly data

### PR DESCRIPTION
**Pull request by lukevella**
Hi, I've updated the information for Rallly as some of it was a bit out of date and it is currently listed in the wrong category.

### CODE REVIEW

File: software/rallly.yml [hunk 1]
```diff
@@ -1,13 +1,16 @@
 name: Rallly
 website_url: https://rallly.co
-description: Create polls to vote on dates and times (alternative to Doodle).
+description: Open-source Doodle alternative. Poll for availability and schedule group meetings.
```
Comment (KieranRobson) [inline software/rallly.yml:3]: Please keep the (alternative to Doodle) as this is standard throughout most of the descriptions

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Quick change to be made

File: software/rallly.yml (hunk 1 above)
Comment (lukevella) [reply software/rallly.yml]: Done 👍

**Review by nodiscc — CHANGES_REQUESTED** [review]
I think the previous description was actually better - if you really think it should be changed, why not use the official description from https://github.com/lukevella/rallly/blob/main/README.md?

> Schedule group meetings with friends, colleagues and teams. Create meeting polls to find the best date and time to organize an event based on your participants' availability.

**Comment by lukevella** [comment]
Sure, I've reverted the change to the description then 👍

File: software/rallly.yml [hunk 2]
```diff
@@ -4,10 +4,13 @@ description: Create polls to vote on dates and times (alternative to Doodle).
 licenses:
   - AGPL-3.0
 platforms:
-  - Nodejs
+  - Node.js
```
Comment (nodiscc) [inline software/rallly.yml:7]:
```suggestion
  - Nodejs
```

The previous syntax was correct https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/platforms/nodejs.yml

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]
Thanks

**Pull request merged**